### PR TITLE
[PagoPA-860] delete bundles by cdi int test

### PR DESCRIPTION
**Pull request by alessio-acitelli**
#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: .github/workflows/code_review.yml [hunk 1]
```diff
@@ -33,15 +33,61 @@ jobs:
 
     # Steps represent a sequence of tasks that will be executed as part of the job
     steps:
-      - name: Code Review
-        uses: pagopa/github-actions-template/maven-code-review@v1.4.2
+      - uses: actions/checkout@v2
```
Comment (jacopocarlini) [inline .github/workflows/code_review.yml:36]: don't change this part. It is the **new** code-review job.

File: integration-test/src/config/.env.local [hunk 2]
```diff
@@ -0,0 +1 @@
+AFM_UTILS_HOST=http://localhost:8586
```
Comment (jacopocarlini) [inline integration-test/src/config/.env.local:1]: use port 8080

File: integration-test/src/step_definitions/support/steps.js [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    responseToCheck = await call(method, afm_utils_host + urlDeleteBundlesByIdCDI);
+  });
+
+Then(/^check statusCode is (\d+)$/, function(status) {
+  assert.strictEqual(responseToCheck.status, status);
+});
+
+// Asynchronous Promise
+AfterAll(async function() {
+  await sleep(1000);
```
Comment (jacopocarlini) [inline integration-test/src/step_definitions/support/steps.js:58]: why?

File: docker/run_docker.sh [hunk 4]
```diff
@@ -26,7 +26,7 @@ if test -f "$FILE"; then
     rm .env
 fi
 config=$(yq  -r '."microservice-chart".envConfig' ../helm/values-$ENV.yaml)
-for line in $(echo $config | jq -r '. | to_entries[] | select(.key) | "\(.key)=\(.value)"'); do
+for line in $(echo $config | jq -r '. | to_entries[] | select(.key != "CONFIGURATION_TRIGGER") | "\(.key)=\(.value)"'); do
```
Comment (jacopocarlini) [inline docker/run_docker.sh:29]: 👍 it's ok

File: integration-test/src/step_definitions/support/datastore_client.js
Comment (jacopocarlini) [inline integration-test/src/step_definitions/support/datastore_client.js:1]: great work

File: integration-test/src/step_definitions/support/steps.js (hunk 3 above)
Comment (alessio-acitelli) [reply integration-test/src/step_definitions/support/steps.js]: except one removed

**Comment by alessio-acitelli** [comment]
update_code

**Pull request merged**